### DISCUSSION TITLE
How to disable display & manage memory?

### DISCUSSION THREAD

**Discussion by lejeczek** [ROCm/ROCm#2898] [Q&A] [answered]
Hi guys.
I've got my first Radeon GPU in a long long time in hope to learn some ML/LLM, _hugginfgace_ specifically - complete novice I'm.
I struggle with tiny 8GB - from perspective of these things - on my RX 6600 and I wonder...
although I realize it's not ROCm specific - would you know how to disable/tweak bits in kernel/module so display is completely disabled and all vRAM if for computing only - I've been running headless all this time anyway.
many thanks, L.

**Comment by zichguan-amd** [comment, marked as answer]
Hi @lejeczek, you can disable the display manager with `systemctl stop display-manager`, or change boot target to `multi-user.target` for CLI like `sudo systemctl isolate multi-user.target` or `sudo systemctl set-default multi-user.target` to change default behaviour. You can also delete the display manager, desktop environment, or display driver.
